src/ClientBuilder.php

```diff
@@ -206,6 +212,10 @@ private function createDefaultTransportFactory(): DefaultTransportFactory
             $this->sdkVersion
         );
 
-        return new DefaultTransportFactory($streamFactory, Psr17FactoryDiscovery::findRequestFactory(), $httpClientFactory, $this->logger);
+        return new DefaultTransportFactory(
+            $streamFactory, Psr17FactoryDiscovery::findRequestFactory(),
+            $httpClientFactory,
+            $this->logger
+        );
```

Comment:
I think this should've been:

```suggestion
        return new DefaultTransportFactory(
            $streamFactory,
            Psr17FactoryDiscovery::findRequestFactory(),
            $httpClientFactory,
            $this->logger
        );
```